JavaKotlinPracticals/app/src/main/java/com/example/javakotlinpracticals/ArrayDemo.kt

```diff
@@ -0,0 +1,42 @@
+package com.example.javakotlinpracticals
+
+fun main()
+{
+    //Arrays Declaration and Intialization
+    val arr = arrayOf("Java","Kotlin","Swift")
+    var arr1 = arrayOf(10,20,30)
+    var arr2 = arrayOf<String>("One","Two","Three")
```

Comment:
Here, it is okay that you have provided arr1, arr2 because it is for learning purpose, but try to provide meaningful name to every variable in code i.e., numbers, languages etc.